### PR TITLE
Refactoring

### LINKED ISSUE #46: Refactoring and Code Cleanup
Many variables names do not follow Dart's guidelines. Various instances of unused functions and imports. A lot of redundant code is present.

### PR DESCRIPTION
**Pull request by Thesmader**
Fixes #46 
Redundant MediaQuery calls removed.
Variable names changed to follow Dart guidelines.

### CODE REVIEW

File: scp/lib/booking.dart [hunk 1]
```diff
@@ -78,7 +80,7 @@ class _BookingState extends State<Booking> {
             child: Text(
               'Confirmation',
               style: TextStyle(
-                  fontSize: queryWidth * 0.065,
+                  fontSize: screenWidth * 0.065,
```
Comment (ankank30) [inline scp/lib/booking.dart]: Are you sure that fontSize in relation to screenWidth is a better option than using `textScaleFactor`?

File: scp/lib/main.dart [hunk 2]
```diff
@@ -299,13 +314,13 @@ class _HomePageState extends State<HomePage> {
...
-      print(e.message);
-    }
-  }
+  // _startFAQActivity() async {
+  //   try {
+  //     await platform.invokeMethod('startFaqActivity');
+  //   } on PlatformException catch (e) {
+  //     print(e.message);
+  //   }
+  // }
```
Comment (ankank30) [inline scp/lib/main.dart]: This comment is intentional, right?

File: scp/lib/booking.dart (hunk 1 above)
Comment (Thesmader) [reply scp/lib/booking.dart]: The Text widget has a property called textScaleFactor and, put simply, the given font size gets multiplied by the textScaleFactor to determine the font size that is actually rendered on the screen in logical pixels. So, if you wanted the text to be 150% of its normal size then you’d set your textScaleFactor to 1.5.
According to me, the textScaleFactor would come in handy while defining layouts for different screens.

File: scp/lib/main.dart (hunk 2 above)
Comment (Thesmader) [reply scp/lib/main.dart]: Yes, it is. The function was unused. FAQ activity works correctly, even without the method. The purpose of that function is still unknown, so it is commented out.

**Pull request merged**